openhtf/io/proto/assembly.proto

```diff
@@ -0,0 +1,103 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
```

Comment:
@fahhem are you talking about adding assembly info support to some part of OpenHTF _besides_ `OutputToTestRun`?